salt/search/init.sls

```diff
@@ -102,13 +109,15 @@ search-ensure-index:
         - require:
             - search-console-ready
             - search-cache-clean
+{% endif %}
 
 # useful for smoke testing the JSON output
 search-jq:
     pkg.installed:
         - pkgs:
             - jq
 
+{% if leader %}
 gearman-db-user:
```

Comment:
Maybe the Gearman states should go in their own file?

Reply:
Could do the same with ElasticSearch?

Reply:
They are, but this is the Gearman persistence. Which could still go into its own file